lib/install/management/store_files.go

```diff
@@ -53,15 +53,37 @@ func (d *Dispatcher) DeleteStores(vchVM *vm.VirtualMachine, conf *metadata.Virtu
 	var emptyImages bool
 	var emptyVolumes bool
 	log.Infof("Removing images")
-	if emptyImages, err = d.deleteImages(ds, p); err != nil {
-		errs = append(errs, err.Error())
+	for _, imageDir := range conf.ImageStores {
```

Comment:
These logic all delete images function related, please move them to deleteImages method.